mhs/common/mhs_common/workflow/synchronous.py

```diff
@@ -69,42 +68,39 @@ def __init__(self,
                                                                                 interaction_details=interaction_details,
                                                                                 message=payload)
         except Exception:
-            logger.error('002', 'Failed to prepare outbound message')
+            logger.error('Failed to prepare outbound message')
             await wdo.set_outbound_status(wd.MessageStatus.OUTBOUND_MESSAGE_PREPARATION_FAILED)
             return 500, 'Failed message preparation', None
 
         if len(message) > self.max_request_size:
-            logger.error('004', 'Request to send to Spine is too large after serialisation. '
-                                '{RequestSize} {MaxRequestSize}',
-                         {'RequestSize': len(message), 'MaxRequestSize': self.max_request_size})
+            logger.error('Request to send to Spine is too large after serialisation. {RequestSize} {MaxRequestSize}',
```

Comment:
`exception()` ?

Reply:
The original logging didn't log exception so I've left it the same